hack/rp-dev/rp_funcs.sh

```diff
@@ -0,0 +1,197 @@
+        make pre-deploy-no-aks
+    else
+        echo "All the 6 deployments exists. ⏩📋 Predeployment was skipped"
+    fi
+    echo -e "Success step 3 ✅ - deploy pre-deployment resources prior to AKS\n"
+}
+
+add_hive(){
+  source hack/devtools/deploy-shared-env.sh
+  check_deployment ${RESOURCEGROUP} dev-vpn && echo "⏩📋 VPN deployment was skipped" \
+  || deploy_vpn_for_dedicated_rp && echo "Success step 4a 🚀 - VPN has been deployed"
+  check_deployment ${RESOURCEGROUP} aks-development && echo "⏩📋 VPN deployment was skipped" \
+  || deploy_aks_dev && echo "Success step 4b 🚀 - AKS has been deployed"
+  echo -e "Success step 4 ✅ - VPN & AKS have been deployed\n"
+
+  vpn_configuration
+  screen -dmS connect_dev_vpn bash -c "sudo openvpn secrets/vpn-$LOCATION.ovpn; sleep 10; exec bash" # open new socket to run concurrently
+  make aks.kubeconfig
+  HOME=/usr ./hack/hive-generate-config.sh # HOME is default to /usr which isn't in PATH
+  KUBECONFIG=$(pwd)/aks.kubeconfig ./hack/hive-dev-install.sh
+  echo -e "Success step 5 ✅ - Hive has been installed\n"
+}
+
+mirror_images() {
+  export DST_ACR_NAME=${AZURE_PREFIX}aro
+  export SRC_AUTH_QUAY=$(echo $USER_PULL_SECRET | jq -r '.auths."quay.io".auth')
+  export SRC_AUTH_REDHAT=$(echo $USER_PULL_SECRET | jq -r '.auths."registry.redhat.io".auth')
+  export DST_AUTH=$(echo -n '00000000-0000-0000-0000-000000000000:'$(az acr login -n $DST_ACR_NAME --expose-token | jq -r .accessToken) | base64 -w0)
+  docker login -u 00000000-0000-0000-0000-000000000000 -p "$(echo $DST_AUTH | base64 -d | cut -d':' -f2)" "$DST_ACR_NAME.azurecr.io"
+  echo "Success step 6a ✈️ 🏷️ - Login to ACR"
+
+  if [[  "${SKIP_DEPLOYMENTS}" == "false" ]] || ! check_acr_repos ${AZURE_PREFIX}-global ; then
+    make go-verify
+    echo "Success step 6b - Add Vendor directory"
+
+    go run -tags containers_image_openpgp,exclude_graphdriver_btrfs ./cmd/aro mirror latest
+    echo "Success step 6c ✈️ 📦 - Mirror OCP images"
+
+    mdm_image_tag=$(get_digest_tag "MdmImage")
+    mdm_repo=$(echo $mdm_image_tag | sed -n 's|.*/\([^/]*\/[^:]*\):.*|\1|p')
+    mdsd_image_tag=$(get_digest_tag "MdsdImage")
+    mdsd_repo=$(echo $mdsd_image_tag | sed -n 's|.*/\([^/]*\/[^:]*\):.*|\1|p')
+    import_geneva_image $mdm_repo $mdm_image_tag
+    import_geneva_image $mdm_repo $mdm_image_tag
+    echo "Success step 6d ✈️ 📦 - Import MDM and MDSD to ACR"
+
+    make publish-image-aro-multistage
+    echo "Success step 6e ✈️ 📦 - Build, push and import to ARO image to ACR"
+
+    fluentbit_image_tag=$(get_digest_tag "FluentbitImage")
+    copy_digest_tag $PULL_SECRET "arointsvc" $DST_ACR_NAME $fluentbit_image_tag
+    echo "Success step 6f ✈️ 📦 - Copy Fluenbit image to ACR"
+  else
+    echo "⏩📋 Skip mirroring repos to ACR $DST_ACR_NAME" 
+  fi
+  echo -e "Success step 6 ✅ - Mirror repos to ACR $DST_ACR_NAME\n"
+}
+
+prepare_RP_deployment() {
+  export PARENT_DOMAIN_NAME=osadev.cloud
+  export GLOBAL_RESOURCEGROUP=${AZURE_PREFIX}-global
```

Comment:
1. Use lowercase names
2. Double quotes
3. Pass in `AZURE_PREFIX` as an arguement, assign to local nameref variable